src/test/groovy/com/github/vkorobkov/jfixtures/loader/RowTest.groovy

```diff
@@ -174,4 +174,60 @@ class RowTest extends Specification {
         def exception = thrown(IllegalArgumentException)
         exception.message == "Parameter <keyValuePairs> is expected to have odd length since it represents key/value pairs"
     }
+
+    def "#column returns a row"() {
```

Comment:
`#column adds a new column to the row`

Reply:
And the similar pattern is for other test methods names

Reply:
thanks